src/app/features/event/event-card/event-card.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="event-card">
+  <div class="event-card__info">
+    <img src="../../../../assets/image 3.png" alt="event image" />
```

Comment:
da się tutaj zrobić taki globalny export w index.ts
export * from './sciezka 
żeby uniknąć takiego zagnieżdżenia? 
albo dodać ścieżkę do zmiennej i ją tutaj przypisać? 
Takie luźne sugestie - to nie przeszkadza, ale lepiej by wyglądało.

Reply:
`src="assets/image_3.png"` powinno zadziałać, btw nie dodajwamy plików ze specjami, zamieńmy `image 3` na `image_3`, ponieważ na linuksie mogą być potencjalne problemy

Reply:
Zmieniłem ścieżkę i poprawiłem nazwę pliku.